tests/integration/conftest.py

```diff
@@ -864,6 +876,30 @@ def get_full_kernel_title(shell, kernel=None):
     return full_title
 
 
+def get_custom_repos_names():
+    """
+    Helper function.
+    Returns a list of correct repo names used on RHEL respecting major/eus system version.
+    """
+    system_version = SystemInformationRelease.version
+
+    # Default RHEL-7 repositories
+    repos = ["rhel-7-server-rpms", "rhel-7-server-optional-rpms", "rhel-7-server-extras-rpms"]
```

Comment:
Is rhel-7-server-supplementary-rpms repo needed to test?

Reply:
I think we have never used this repository for testing. It can be added. Have you seen it specified somewhere in the documentation?

Reply:
I'm going to merge this PR, we can enable another repository in the following PRs